packages/dev/inspector-v2/src/services/panes/properties/atmospherePropertiesService.tsx

```diff
@@ -0,0 +1,63 @@
+import type { ServiceDefinition } from "../../../modularity/serviceDefinition";
+import type { ISelectionService } from "../../selectionService";
+import type { IPropertiesService } from "./propertiesService";
+import type { ISceneContext } from "../../../services/sceneContext";
+import { SelectionServiceIdentity } from "../../selectionService";
+import { PropertiesServiceIdentity } from "./propertiesService";
+import { SceneContextIdentity } from "../../../services/sceneContext";
+import { Atmosphere } from "addons/atmosphere/atmosphere";
+import {
+    AerialPerspectiveProperties,
+    DiffuseIrradianceProperties,
+    GeneralAtmosphereProperties,
+    MultipleScatteringProperties,
+    RenderingOptionsProperties,
+    ScatteringAndAbsorptionProperties,
+} from "../../../components/properties/atmosphereProperties";
+export const AtmospherePropertiesServiceDefinition: ServiceDefinition<[], [IPropertiesService, ISelectionService, ISceneContext]> = {
+    friendlyName: "Atmosphere Properties",
+    consumes: [PropertiesServiceIdentity, SelectionServiceIdentity, SceneContextIdentity],
+    factory: (propertiesService, selectionService, sceneContext) => {
+        const scene = sceneContext.currentScene;
+        if (!scene) {
+            return undefined;
+        }
+
+        const atmosphereContentRegistration = propertiesService.addSectionContent({
+            key: "Atmosphere Properties",
+            predicate: (entity: unknown) => entity instanceof Atmosphere,
+            content: [
+                {
+                    section: "General",
+                    component: ({ context }) => <GeneralAtmosphereProperties entity={context} />,
+                },
+                {
+                    section: "Scattering And Absorption",

```

Comment:
Tiny, but for Title Case it should actually be
```suggestion
                    section: "Scattering and Absorption",
```